src/Dfe.PlanTech.Web/Controllers/CheckAnswersController.cs

```diff
@@ -28,18 +35,23 @@ public async Task<IActionResult> CheckAnswersPage([FromServices] ProcessCheckAns
             SectionName = parameterCheckAnswersPage.SectionName,
             CheckAnswerDto = await processCheckAnswerDtoCommand.ProcessCheckAnswerDto(parameterCheckAnswersPage.SubmissionId, parameterCheckAnswersPage.SectionId),
             Content = checkAnswerPageContent.Content,
-            SubmissionId = parameterCheckAnswersPage.SubmissionId
+            SectionSlug = parameterCheckAnswersPage.SectionSlug,
+            SubmissionId = parameterCheckAnswersPage.SubmissionId,
+            Slug = checkAnswerPageContent.Slug
         };
 
         return View("CheckAnswers", checkAnswersViewModel);
     }
 
     [HttpGet]
-    [Route("change-answer")]
-    public IActionResult ChangeAnswer(string questionRef, string answerRef, int submissionId)
+    [Route("change-answer", Name = "ChangeAnswerRouteLink")]
```

Comment:
Does this need to be named?

Reply:
Yes, the name is there so I can differentiate the different pages, so it doesn't route to the wrong page.  This is also how we can have cleaner urls.